client/build/get_build_parameters.mjs

```diff
@@ -33,6 +33,7 @@ export function getBuildParameters(cliArguments) {
     verbose = false,
     versionName = '0.0.0',
     sentryDsn = process.env.SENTRY_DSN,
+    arch = 'default',
```

Comment:
This `'default'` string appears in multiple files and will be hard to maintain, probably just use `null` or empty string`''`? Then in build.action.mjs, you can simply concatenate it to the platform without any conditions.

Reply:
done